src/components/DefinitionList/DefinitionList.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+
+import {ClipboardButton, Link} from '@gravity-ui/uikit';
+
+import {HelpPopover} from '../HelpPopover';
+import type {HelpPopoverProps} from '../HelpPopover';
+import {block} from '../utils/cn';
+
+import {isUnbreakableOver} from './utils';
+
+import './DefinitionList.scss';
+
+type DefinitionListItemNote = string | HelpPopoverProps;
+
+export interface DefinitionListItem {
+    name: string;
+    key?: string;
+    href?: string;
+    content?: React.ReactNode;
+    title?: string;
+    copyText?: string;
+    copyIconOver?: boolean;
+    note?: DefinitionListItemNote;
+    multilineName?: boolean;
+}
+
+export interface DefinitionListProps {
+    items: DefinitionListItem[];
+    responsive?: boolean;
+    keyMaxWidth?: number;
+    valueMaxWidth?: number | 'auto';
+    className?: string;
+    itemClassName?: string;
+}
+
+export const b = block('definition-list');
+
+function getTitle(title?: string, content?: React.ReactNode) {
+    if (title) {
+        return title;
+    }
+
+    if (typeof content === 'string' || typeof content === 'number') {
+        return String(content);
+    }
+
+    return undefined;
+}
+
+function getNoteElement(note?: DefinitionListItemNote) {
+    let noteElement = null;
+    const popoverClassName = b('item-note-tooltip');
+    if (note) {
+        if (typeof note === 'string') {
+            noteElement = (
+                <HelpPopover
+                    className={popoverClassName}
+                    tooltipContentClassName="yfm"
+                    htmlContent={note}
+                    placement={['bottom', 'top']}
+                />
+            );
+        }
+
+        if (typeof note === 'object') {
+            noteElement = <HelpPopover className={popoverClassName} {...note} />;
+        }
+    }
+    return noteElement;
+}
+
+export function DefinitionList({
+    items,
+    responsive,
+    keyMaxWidth,
+    valueMaxWidth = 'auto',
+    className,
+    itemClassName,
+}: DefinitionListProps) {
+    const keyStyle = keyMaxWidth
+        ? {
+              flexBasis: keyMaxWidth,
+          }
+        : {};
+
+    const valueStyle =
+        typeof valueMaxWidth === 'number'
+            ? {
+                  flexBasis: valueMaxWidth,
+                  maxWidth: valueMaxWidth,
+              }
+            : {};
+    return (
+        <dl className={b({responsive}, className)} role="list">
```

Comment:
role non needed, dl has own role

Reply:
It's like assigning role="button" to `<button>` element. The problem is with the tests not markup